packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/location-listing.twig

```diff
@@ -9,7 +9,9 @@
   {% endif %}
   <div class="ma__location-listing__columns">
     <div class="ma__location-listing__container">
-      <div class="ma__location-listing__map">
+      <div class="ma__location-listing__map" aria-label="{{leafletMap.title}}">
+        <a href="#location-listing-results" aria-describedby="sr-info">skip to the location listing</a>
```

Comment:
```suggestion
        <a href="#location-listing-results" aria-describedby="sr-info">skip to the location listing</a>
```
This should be a real id that's passable and the ID should be placed in the html of the listing div.
I can see this link, be nothing is happening when it's clicked or `selected.```